base/errorshow.jl

```diff
@@ -543,7 +548,7 @@ function show_method_candidates(io::IO, ex::MethodError, @nospecialize kwargs=()
 
     if !isempty(lines) # Display up to three closest candidates
         Base.with_output_color(:normal, io) do io
-            print(io, "\nClosest candidates are:")
+            print(io, "\n\nClosest candidates are:")
```

Comment:
```suggestion
            print(io, "\nClosest candidates are:")
```
Fewer lines is better IMO. Especially with long stacktraces.

Reply:
This was expressly added to seperate the blocks visually and to make it easier to read - removing them is IMO very counter to what I'm trying to achieve here.

Reply:
For comparison, here's what the current PR would look like with those newlines removed:

<details>
<summary> Click for screenshot </summary>

![screenshot_2022-04-24_15-44-31_570294865](https://user-images.githubusercontent.com/11753998/164979531-460b89b9-bdb0-400a-883b-8059ac1c6b1b.png)

</details>

IMO that is harder to read, because it's all one big blob of text, even with the color change.